txserver/handler.go

```diff
@@ -39,7 +39,28 @@ var handlerMap = map[string]func(*context.Context, *Command) ([]byte, error){
 }
 
 func add(ctx *context.Context, command *Command) ([]byte, error) {
-	return []byte{}, nil
+	account, err := find_account(ctx, command.Username)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
One thing we need to do is create a user account if account not found. I'll create a fn for that